### PR TITLE
Feat add user reg logger

### PR DESCRIPTION
**Pull request by eesaanatluri**
This PR consists of agent that stores the account request object inside a sqlite DB.

### CODE REVIEW

**Review by rtripath89 — CHANGES_REQUESTED** [review]

File: user_reg_logger.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+import json
+import sys
+import dataset
+import logging
```
Comment (diedpigs) [inline user_reg_logger.py:5]: Once you import `rc_util`, you don't have to import `logging`

File: user_reg_logger.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+from datetime import datetime
+
+# Define queue name 
+task = 'reg_logger'
+
+# Instantiate rabbitmq object
+reg_logger = RCRMQ({'exchange': 'RegUsr', 'exchange_type': 'topic'})
+
+# Parse arguments
+args = rc_util.get_args()
```
[the remark this answers is not in the document]
Comment (eesaanatluri) [reply user_reg_logger.py:22]: Done

File: user_reg_logger.py (hunk 1 above)
Comment (eesaanatluri) [reply user_reg_logger.py]: Done

File: user_reg_logger.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (eesaanatluri) [reply user_reg_logger.py:16]: Done Could you check now ?

**Pull request merged**